build/yamls/antrea-aks.yml

```diff
@@ -1367,6 +1367,10 @@ data:
     # whenever a Pod's container defines a specific port to be exposed (each container can define a list of ports as pod.spec.containers[].ports),
     # and all Node traffic directed to that port will be forwarded to the Pod.
     #nplPortRange: 40000-41000
+
+    # Provide the address of Kubernetes apiserver, to override any value provided in kubeconfig or InClusterConfig.
+    # Defaults to "". It must be a host string, a host:port pair, or a URL to the base of the apiserver.
```

Comment:
Maybe we could remove "to the base of the apiserver", if this sentence is about the format of the value.

Reply:
"a URL to the base of apiserver" is together, which means the path of the url should be excluded (like `https://10.96.0.1` not `https://10.96.0.1/apis/v1`). Actually I copied it from client-go.

Reply:
Sure.